docs/reference/builder.md

```diff
@@ -1222,6 +1222,12 @@ This Dockerfile results in an image that causes `docker run`, to
 create a new mount point at `/myvol` and copy the  `greeting` file
 into the newly created volume.
 
+> **Warning**:
+> When using Windows-based containers, the destination of a volume or
+> bind-mount inside the container must be one of: a non-existing or empty
+> directory; or a drive other than C:. Further, the source of a bind mount must
+> be a local directory, not a file.
+
```

Comment:
I ripped this text off of here: https://github.com/docker/docker/blob/master/docs/reference/commandline/run.md#mount-volume--v---read-only

Original read:

> The following examples will fail when using Windows-based containers, as the destination of a volume or bind-mount inside the container must be one of: a non-existing or empty directory; or a drive other than C:. Further, the source of a bind mount must be a local directory, not a file.